common/v2/features/Dashboard/components/AccountDropdown.tsx

```diff
@@ -60,22 +61,31 @@ const SDropdown = dropdown`
 
 const renderAccounts = (
   accounts: ExtendedAccount[],
+  labels: AddressBook[],
   selected: string[],
   handleChange: (uuid: string) => void
 ) =>
-  accounts.map(({ uuid, label, address }: ExtendedAccount) => (
-    <Checkbox
-      key={uuid}
-      name={`account-${uuid}`}
-      checked={selected.includes(uuid)}
-      onChange={() => handleChange(uuid)}
-      label={label}
-      icon={() => <Identicon className="AccountDropdown-menu-identicon" address={address} />}
-    />
-  ));
+  accounts.map(({ uuid, address }: ExtendedAccount) => {
+    const detectedLabel: AddressBook | undefined = labels.find(
+      label => label.address.toLowerCase() === address.toLowerCase()
+    );
+    const addressLabel = !detectedLabel ? 'Unknown Account' : detectedLabel.label;
```

Comment:
Avoid the negation in a ternary (harder to reason). It's simpler to just invert the expressions.
```
 const addressLabel = detectedLabel ? detectedLabel.label :  'Unknown Account' ;
```